docs/install-config/harbor-compatibility-list.md

```diff
@@ -48,6 +48,8 @@ This document provides compatibility information for all Harbor components.
 | [DoSec](https://github.com/dosec-cn/harbor-scanner/blob/master/README_en.md)|![DoSec](../../img/scanners/dosec.png)    | DoSec    | ![Y](../../img/replication-adapters/right.png) | N | v1.10 |
 | [Sysdig Secure](https://github.com/sysdiglabs/harbor-scanner-sysdig-secure) |![Sysdig](../../img/scanners/sysdig.png) | Sysdig  | ![Y](../../img/replication-adapters/right.png) | N | v2.1.0 |
 | [TensorSecurity](https://github.com/tensorsecurity/harbor-scanner) |![TensorSecurity](../../img/scanners/tensorsecurity.png) | TensorSecurity | ![Y](../../img/replication-adapters/right.png) | N | v2.2.0 |
+| [ArksecScanner](https://github.com/arksec-cn)    |![Arksec](../../img/scanners/arksec.png)| Arksec    |![Y](../../img/replication-adapters/right.png)|N| v2.4.0 |
```

Comment:
@simonsuns I suppose that should be also tested by someone from the maintainers team to verify that it's working as expected and introducing some security or performance issues! @goharbor/all-maintainers anyone ?

Reply:
@wy65701436 @yanji09 - please check about since I'm not sure we have to verify that or we just replay on 3rd party verification?

Reply:
@heww @steven-zou would you take a look at this? (arcsec did a demo in one of our community meeting.)

Reply:
Maybe we need help from @zyyw. 

Here is their scanner repo URL: https://github.com/arksec-cn/harbor-scanner

Reply:
it's looks good to me by adding this arksec scanner in Harbor and getting some scanning reports back.
<img width="1778" alt="Screen Shot 2021-12-24 at 11 22 35 AM" src="https://user-images.githubusercontent.com/6709992/14732446
<img width="1787" alt="Screen Shot 2021-12-24 at 11 22 54 AM" src="https://user-images.githubusercontent.com/6709992/147324499-da10434c-3e5c-4460-830f-227fc757eee0.png">
8-b5dfa4b8-63ea-4fd4-85a8-749dec646e18.png">